src/components/ValidatedInput/ValidFieldText.tsx

```diff
@@ -50,6 +50,7 @@ export interface ValidFieldTextProps
   dataQaProps?: string | null;
   validFieldTextRef?: React.RefObject<HTMLDivElement>;
   rightIcon?: React.ReactNode;
+  /** @deprecated Use type="float" instead */
   isFloat?: boolean;
```

Comment:
Is there any field using the `isFloat` props right now? I don't find any in AP master, how about delete it here?

Reply:
okay

Reply:
done